docs: add graphql-tag-swc-plugin to performance docs (#11884)
apollographql/apollo-client: docs/source/performance/babel.md

And that's it! All the usages of `import gql from 'graphql-tag'` will be removed, and the calls to `gql` will be replaced by the compiled version.

## Using graphql-tag-swc-plugin

> This plugin is same as [babel-plugin-graphql-tag](#using-babel-plugin-graphql-tag) but for swc, a fast rust based babel alternative.

Reviewer: ```suggestion
This plugin is same as [babel-plugin-graphql-tag](#using-babel-plugin-graphql-tag) but for [swc](https://swc.rs/), a fast rust based babel alternative.
```

Might be helpful to link out to `swc`'s website :)

Install the plugin in your dev dependencies:

```
# with npm
npm install --save-dev graphql-tag-swc-plugin

# or with yarn
yarn add --dev graphql-tag-swc-plugin
```

Then add the plugin in your `.swcrc` configuration file:

```
{
  jsc: {
    experimental: {
      plugins: [
        ["graphql-tag-swc-plugin",
          {
            importSources: ["@apollo/client", "graphql-tag"],
            gqlTagIdentifiers: ["gql"]
          },
        ],
      ],
    },
  },
}
```

> For more information on configuration and features, please checkout [`graphql-tag-swc-plugin` here](https://github.com/rishabh3112/graphql-tag-swc-plugin).

Reviewer: ```suggestion
For more information on configuration and features, please checkout the [`graphql-tag-swc-plugin`](https://github.com/rishabh3112/graphql-tag-swc-plugin) documentation.
```

}
```

> For more information on configuration and features, please checkout the [`graphql-tag-swc-plugin` here](https://github.com/rishabh3112/graphql-tag-swc-plugin) documentation.

Reviewer: ```suggestion
For more information on configuration and features, please checkout the [`graphql-tag-swc-plugin`](https://github.com/rishabh3112/graphql-tag-swc-plugin) documentation.
```
